Python/bytecodes.c

```diff
@@ -206,7 +206,13 @@ dummy_func(
 
         inst(LOAD_FAST_CHECK, (-- value)) {
             value = GETLOCAL(oparg);
-            ERROR_IF(value == NULL, unbound_local_error);
+            if (value == NULL) {
+                _PyEval_FormatExcCheckArg(tstate, PyExc_UnboundLocalError,
+                    UNBOUNDLOCAL_ERROR_MSG,
+                    PyTuple_GetItem(_PyFrame_GetCode(frame)->co_localsplusnames, oparg)
+                );
+                ERROR_IF(1, error);
```

Comment:
Is there some reason this (and the others with this same pattern) can't be `ERROR_NO_POP()`?

```suggestion
                ERROR_NO_POP();
```

Reply:
If there are no input values, then `ERROR_IF` and `ERROR_NO_POP` are the same.

However, `ERROR_NO_POP` implies that there are some input values but that they aren't popped in case of error, which feels like it would be a bit misleading here.